src/GridCal/Engine/Simulations/PowerFlow/power_flow_worker.py

```diff
@@ -434,7 +434,7 @@ def outer_loop_power_flow(circuit: SnapshotData, options: PowerFlowOptions,
     return results
 
 
-def power_flow_post_process(calculation_inputs: SnapshotData, Sbus, V, branch_rates, Yf=None, Yt=None,
+def power_flow_post_process(calculation_inputs: SnapshotData, Sbus, V, branch_rates, Ybus=None, Yf=None, Yt=None,
```

Comment:
If this call changes, all of it's other calls need to change as well

Reply:
Yep, I finally decided to create the `short_circuit_worker.py` and write the `short_circuit_post_process` function, which is quite similar to the `power_flow_post_process` function. In my opinion it is a bit cleaner this way. (See commit 59a8ad7).